### ISSUE TITLE
Vulnerability detected in 7.0.0

### ISSUE THREAD

**Issue by eaviles** [sindresorhus/eslint-plugin-unicorn#210]
Thanks for the v7.0.0 release. I tried to use it but Snyk detected a vulnerability that made me stop. It was detected through:

eslint-plugin-unicorn@7.0.0 › safe-regex@2.0.1 › regexp-tree@0.0.86 › yargs@10.1.2 › os-locale@2.1.0 › mem@1.1.0

And here are more details about it:
https://app.snyk.io/vuln/npm:mem:20180117

I'm yet to try to find a way around it but just wanted to bring it to your attention.

**Comment by eaviles** [comment]
> The vulnerability can only possibly affects servers, so the report is not relevant for `eslint-plugin-unicorn`.

That's correct @sindresorhus but many projects have the vulnerability checks in their builds that also include development dependencies. All our own project builds in my company started failing because of this, we are heavy users of eslint-plugin-unicorn but we might stay in version 6 until resolved.

**Comment by teppeis** [comment]
I've made a PR to fix this issue: https://github.com/DmitrySoshnikov/regexp-tree/pull/171

**Comment by teppeis** [comment]
next: https://github.com/davisjam/safe-regex/pull/22

**Comment by teppeis** [comment]
regexp-tree and safe-regex was updated. Now unicorn is not marked as vulnerable.
https://app.snyk.io/test/npm/eslint-plugin-unicorn

**Comment by MrHen** [comment]
Thank you @teppeis!

**Comment by eaviles** [comment]
Thanks a lot @tappeis! Highly appreciated it.
